pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -70,6 +70,29 @@ func TestClusterWideMode(t *testing.T) {
 					"expected SMMR to show 50 member namespaces, but that wasn't the case"))
 		})
 
+		t.NewSubTest("RoleBindings verification").Run(func(t test.TestHelper) {
+			t.Log("Related to OSSM-3468")
+			t.LogStep("Check that Rolebindings are not created in the member namespaces")
+			retry.UntilSuccess(t, func(t test.TestHelper) {
+				oc.Get(t, "member-0", "rolebindings", "",
+					assert.OutputContains("system:deployers",
+						"The Rolebings contains system:deployers role",
+						"The Rolebings does not contains system:deployers role"),
+					assert.OutputContains("system:image-builders",
+						"The Rolebings contains system:image-builders role",
+						"The Rolebings does not contains system:image-builders role"),
+					assert.OutputContains("system:image-pullers",
+						"The Rolebings contains system:image-pullers role",
+						"The Rolebings does not contains system:image-pullers role"),
```

Comment:
Why check these rolebindings? They're not related to OSSM, so I think we should ignore them.